Resources/Private/Fusion/Shame.fusion

```diff
@@ -0,0 +1,3 @@
+##
+# Quick & Dirty fixes, that can be improved later on
+#
```

Comment:
haha well it was half a joke half a really useful way to organize code on real projects under pressure. I borrowed this practice from CSS world: https://csswizardry.com/2013/04/shame-css/
I don't insist on adding it to the demo site, but I still believe it's a good thing to teach people to differentiate between quick-n-dirty hacks and the code that's here to stay.

On the contrary, I think the idea of `Override.fusion` is a must-have as a common convention, otherwise globalness of Fusion will kill us.